drivers/azure/azureutil/azureutil.go

```diff
@@ -587,6 +626,20 @@ func (a AzureClient) GetPrivateIPAddress(resourceGroup, name string) (string, er
 	return to.String((*nic.Properties.IPConfigurations)[0].Properties.PrivateIPAddress), nil
 }
 
+func (a AzureClient) GetFQDN(resourceGroup, name string) (string, error) {
```

Comment:
I removed the call, was thinking of leaving it there for future usage, if required. I can remove, if you feel strongly about it.